src/points/types.ts

```diff
@@ -0,0 +1,25 @@
+export enum PointsActivities {
+  CreateWallet = 'CreateWallet',
+  Swap = 'Swap',
+  MoreComing = 'MoreComing',
+}
```

Comment:
nit: I'm on a war against TS enums :D https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant
```suggestion
export type PointsActivity = 'CreateWallet' | 'Swap' | 'MoreComing',
```

Reply:
Oh interesting... I've implemented this, but this actually introduces a new complexity re: testing whether or not a string value is a member of this union at runtime (which we want to do to ensure we don't try to render an activity that doesn't exist).

Reply:
Ah sorry if you need a runtime check too then an enum is warranted 👍

Reply:
It's actually not too bad, we just need a simple typeguard, which isn't much more difficult than checking e.g. `Object.values(SomeEnum)`, but something to keep in mind!